testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/group/JGroupsChannelRuntimeOperationsTestCase.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.jboss.as.test.clustering.cluster.group;
+
+import java.io.IOException;
+import java.time.Duration;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.TargetsContainer;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.as.arquillian.container.ManagementClient;
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.test.clustering.cluster.AbstractClusteringTestCase;
+import org.jboss.dmr.ModelNode;
+import org.jboss.logging.Logger;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.jboss.as.controller.PathAddress.pathAddress;
+import static org.jboss.as.controller.PathElement.pathElement;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.OUTCOME;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.RESULT;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SUCCESS;
+import static org.jboss.as.controller.operations.common.Util.getReadAttributeOperation;
+
+@RunWith(Arquillian.class)
+public class JGroupsChannelRuntimeOperationsTestCase extends AbstractClusteringTestCase {
+    private static final long TIMEOUT_SECONDS = 10;
```

Comment:
TimeoutUtil.adjust(..) 

but this needs to be stabilized anyway, this is not going to fix it on it's own.